Gemfile

```diff
@@ -7,15 +7,15 @@ gem 'rails', '4.2.1'
 gem 'arel'
 gem 'jquery-rails'
 
-gem 'sprockets-rails', '3.0.0.beta1'
```

Comment:
Why is this needed? Last I remember, latest sprockets-rails is actually needed for `sprockets-es6`